Sources/CrashReportConverter.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  CrashReportConverter.swift
+//  SentrySwift
+//
+//  Created by Daniel Griesser on 05/11/16.
+//
+//
+
+import Foundation
+
+public class CrashReportConverter {
+    
+    internal func convertReportToEvent(_ report: CrashDictionary) -> Event? {
```

Comment:
Not a huge this but this could probably just be a static function on a struct since there is no need to initialize an instance